packages/sw-build/src/lib/generate-sw.js

```diff
@@ -43,6 +43,9 @@ const errors = require('./errors');
  * @param {String} [input.cacheId] An optional ID to be prepended to caches
  * used by sw-build. This is primarily useful for local development where
  * multiple sites may be served from `http://localhost`.
+ * @param {Boolean} [input.clientsClaim] An optional boolean that indicates if
+ * the page should skip waiting and claim current pages on a new service worker
```

Comment:
The "skip waiting" bit in the JSDoc isn't accurate. That describes what happens if you call `self.skipWaiting()` inside of an `install` handler, and is independent of what `self.clients.claim()` does.

Some proposed wording:

"...indicates whether the newly activated service worker should take control of existing client pages, including pages already controlled by an older service worker."